src/extras/usdImagingLite/pch.h

```diff
@@ -0,0 +1,176 @@
+/**********************************************************************
```

Comment:
Can we get this file from the USD dir?

Reply:
This file is required and must be in project